api/envoy/api/v2/rds.proto

```diff
@@ -53,7 +61,10 @@ message RouteConfiguration {
   string name = 1;
 
   // An array of virtual hosts that make up the route table.
-  repeated route.VirtualHost virtual_hosts = 2 [(gogoproto.nullable) = false];
+  repeated route.VirtualHost virtual_hosts = 2; [(gogoproto.nullable) = false];
+  // An array of virtual hosts will be dynamically loaded via the VHDS API.
+  // [#not-implemented-hide:]
+  Vhds vhds = 9;
```

Comment:
What's the advantage of this? I'm thinking if VHDS is enabled, then resources will be `vhds`. If VHDS is disabled, we can leave the proto as is and have it use the virtual_hosts field.

Reply:
Nvm. This is the config source.

Reply:
For the sake of posterity: this field is used to configure VHDS protocol (what server to use and such). The actual VHDS exchange uses DeltaRequest/Reponse with ```VirtualHost```s wrapped in ```Resource```s.